primitiv/c/graph.cc

```diff
@@ -7,224 +7,189 @@
 #include <vector>
 
 #include <primitiv/graph.h>
-
 #include <primitiv/c/internal.h>
 #include <primitiv/c/graph.h>
 
 using primitiv::Node;
 using primitiv::Graph;
+using primitiv_c::internal::to_c;
+using primitiv_c::internal::to_cc;
```

Comment:
`to_c` and `to_cc` are very short and similar names.
I think these names can bring bugs by typing errors and lack of understanding.

Suggestion: `to_c_ptr` and `to_cpp_ptr`